src/_posts/2025-08-08-employee-spotlight-adam.md

```diff
@@ -0,0 +1,58 @@
+
+### How have you grown professionally while on our team?
+
+Since starting at Compiler I have learned so much! My mind has exploded with transit! From all the acronyms, to even just behind the scenes work of what makes the wheels go 'round, I have appreciated everything I've learned. Transit was an interest and a hobby of mine, specifically the politics behind transit expansion, but now seeing the nuts and bolts of the technology that goes into everything once it's implemented has been eye opening.
+
+### Which accomplishments are you most proud of?
+
+In 2016, I created a map of the Los Angeles MetroRail and "expanded" the map to show the dreams of what our system could look like if the sales tax measure slated to be on the ballot would pass that Fall. I placed my image in tandem with the current network and created a "blinking gif." That gif went viral! It was written up in local newspapers and I was even interviewed by NPR in regards to the traction it was seeing. Once that sales tax measure passed, I was contacted by the then Mayor's office and asked if I'd be interested in a potential job with them. Having _just_ started on a hit television show, I turned them down... Over the next four years I learned many lessons about life and the world around me, and after all that I still ended up working in transit! Some things truly are just meant to be.
+
+<img src="/assets/blog/2025/2025-08-adam-la-metrorail-measure-m.jpg" alt="">
+
+### How do you like to spend your free time?
+
+If I have any free time, you can catch me on the Los Angeles MetroRail. When I first moved to LA I was so desperate to discover big city life, an escape from my Texan suburban roots. I discovered the Metro Red Line was my ticket from the San Fernando Valley all the way into Hollywood and beyond, into Downtown LA. I've hit the pavement walking all across Los Angeles just discovering new corners, small businesses, new cultures and cuisines. Anyone is welcome to join me any day!
+
+### What’s a cause that you are passionate about?
+
+EXPANDING TRANSIT! It is my number one passion, let's be honest. I truly believe there are so many social causes that are remedied by social programs, and the ability to expand those programs begins and ends with transit. I believe wealth inequality is the greatest plague to mankind, and providing access to jobs and culture to all can physically bridge the gap between the rich and the poor.
+
+<img src="/assets/blog/2025/2025-08-adam-1.jpg" alt="Photo of Adam Linder.">
```

Comment:
Need review of this alt text. Also, @laneymangan you mentioned maybe having a caption on this photo

Reply:
We also need a photo credit to go in the caption.

Reply:
Caption (can also be used as Alt text if needed, otherwise maybe "photo of Adam Linder outside of a Metro station wearing a 'Cars Ruin Cities' shirt and a 'LA' hat"): The photo that started it all! This photo of Adam in his 'Cars Ruin Cities' shirt was taken by Compiler Founder, Vyki Englert, right before the two of them met for the first time. She liked his shirt, told him so, and the rest is history! 

Photo Credit: Vyki Englert